lib/io/rdf/sparql/sparql-query-rdf-source.ts

```diff
@@ -0,0 +1,12 @@
+import {HttpFetch} from "../../fetch/fetch-api";
+import {RdfMemorySource} from "../rdf-source-base";
+import {fetchRdfQuads} from "../http/http-adapter";
+
+export class SparqlQueryRdfSource extends RdfMemorySource {
+
+  async query(httpFetch: HttpFetch, endpoint: string, query: string): Promise<void> {
+    const url = endpoint + "?format=" + encodeURIComponent("text/turtle") + "&query=" + encodeURIComponent(query);
```

Comment:
I would suggest to allow user to set the format, as we have encountered some situations where we need to set text/plain to force TriG as turtle serialization was not valid. With this option hard-coded we would not be able to address such issues.